docs/user-guide.md

```diff
@@ -28,8 +28,8 @@ published: True
       - [NFS share](#nfs-share)
     - [Validating Media](#validating-media)
   - [Prerequisite configuration](#prerequisite-configuration)
-    - [Data mount configuration file](#data-mount-configuration-file)
-    - [ASM disk group configuration file](#asm-disk-group-configuration-file)
+    - [Data mounts configuration](#data-mounts-configuration)
```

Comment:
Nit:  as it's possible to configure a single data mount, "Data mount configuration" would technically be more correct.